subprojects/docs/src/docs/userguide/installation.adoc

```diff
@@ -15,45 +15,148 @@
 
+===== Linux & MacOS users
+
+Configure your `PATH` environment variable to include the `bin` directory of the unzipped distribution, e.g.:
+
+----
+❯ export PATH=$PATH:/opt/gradle/gradle-4.6/bin
+----
+
+===== Microsoft Windows users
+
+In **File Explorer** right-click on the `This PC` (or `Computer`) icon, then click `Properties` -> `Advanced System Settings` -> `Environmental Variables`.
+
+Under `System Variables` select `Path`, then click `Edit`. Add an entry for `C:\Gradle\gradle-4.6\bin`. Click OK to save.
+
+<<sec:installation_next_steps,↓ Proceed to next steps>>
+
+
 [[sec:running_and_testing_your_installation]]
-=== Running and testing your installation
+=== Verifying installation
+
+Open a console (or a Windows command prompt) and run `gradle -v` to run gradle and display the version, e.g.:
+
+----
+❯ gradle -v
+
+------------------------------------------------------------
+Gradle 4.6
+------------------------------------------------------------
+
+Build time:   2018-02-21 15:28:42 UTC
+Revision:     819e0059da49f469d3e9b2896dc4e72537c4847d
+
+Groovy:       2.4.12
+Ant:          Apache Ant(TM) version 1.9.9 compiled on February 2 2017
+JVM:          1.8.0_151 (Oracle Corporation 25.151-b12)
+OS:           Mac OS X 10.13.3 x86_64
+----
+
+If you run into any trouble, see the <<troubleshooting,section on troubleshooting installation>>.
 
-You run Gradle via the `gradle` command. To check if Gradle is properly installed just type `gradle -v`. The output shows the Gradle version and also the local environment configuration (Groovy, JVM version, OS, etc.). The displayed Gradle version should match the distribution you have downloaded.
+You can verify the integrity of the Gradle distribution by downloading the SHA-256 file (available from the link:https://gradle.org/releases[releases page]) and following these <<sec:verification,verification instructions>>.
 
-[[sec:jvm_options]]
-=== JVM options
+[[sec:installation_next_steps]]
+=== Next steps
 
-JVM options for running Gradle can be set via environment variables. You can use either `GRADLE_OPTS` or `JAVA_OPTS`, or both. `JAVA_OPTS` is by convention an environment variable shared by many Java applications. A typical use case would be to set the HTTP proxy in `JAVA_OPTS` and the memory options in `GRADLE_OPTS`. Those variables can also be set at the beginning of the `gradle` or `gradlew` script.
+Now that you have Gradle installed, use these resources for getting started:
 
-Note that it's not currently possible to set JVM options for Gradle on the command line.
+ * Create your first Gradle project by following the link:https://guides.gradle.org/creating-new-gradle-builds/[Creating New Gradle Builds] tutorial.
+ * Sign up for a link:https://gradle.org/training/intro-to-gradle/[live Gradle introduction] with a Gradle core engineer.
```

Comment:
Should this maybe say "live Gradle introduction training"?

Reply:
maybe... I guess I can add that word